openedx/features/partners/give2asia/views.py

```diff
@@ -163,13 +151,15 @@ def create_account_with_params_custom(request, params, partner):
         except Exception as err:  # pylint: disable=broad-except
             log.exception("partner_user creation failed for user {id}, partner {slug}"
                           .format(id=user.id, slug=partner.slug), repr(err))
+            raise
```

Comment:
I think this will cause error instead of raising exception 
`TypeError: exceptions must be old-style classes or derived from BaseException, not NoneType`

Reply:
I have tested this, it raise exception, it does not cause error